MSTest.Sdk: document EnablePlaywright and EnableAspireTesting (#40774)
dotnet/docs: docs/core/testing/unit-testing-mstest-sdk.md

</Project>
```

## Features

Outside of the selection of the runner and runner specific extensions, `MSTest.Sdk` also provides additional features to simplify and enhance your testing experience.

> ### Aspire

Reviewer: ```suggestion
### .NET Aspire
```

> `.NET Aspire` is an opinionated, cloud ready stack for building observable, production ready, distributed applications. .NET Aspire is delivered through a collection of NuGet packages that handle specific cloud-native concerns. For more information about it, check out its [documentation page](https://learn.microsoft.com/dotnet/aspire/get-started/aspire-overview).

Reviewer: ```suggestion
.NET Aspire is an opinionated, cloud ready stack for building observable, production ready, distributed applications. .NET Aspire is delivered through a collection of NuGet packages that handle specific cloud-native concerns. For more information, see the [.NET Aspire docs](/dotnet/aspire/get-started/aspire-overview).
```

Author: Oooooh! I was looking on disk and I couldn't find this path that's why I used the full URL.

> [!NOTE]
> This feature is available from MSTest.Sdk 3.4.0

By setting the property `EnableAspireTesting` to `true` you can bring all dependencies and default usings you would need for testing with `Aspire` and `MSTest`.

```xml
<Project Sdk="MSTest.Sdk/3.4.0">

    <PropertyGroup>
        <TargetFramework>net8.0</TargetFramework>
        <EnableAspireTesting>true</EnableAspireTesting>
    </PropertyGroup>

    <!-- references to the code to test -->

</Project>
```

### Playwright

> `Playwright` enables reliable end-to-end testing for modern web apps. For more information about it, check out its [documentation page](https://playwright.dev/dotnet/docs/intro).

Reviewer: ```suggestion
Playwright enables reliable end-to-end testing for modern web apps. For more information, see the official [Playwright docs](https://playwright.dev/dotnet/docs/intro).
```
